scheduler.py

```
@@ -91,11 +91,12 @@ def add_circuit_job(self, circuit, circuit_scheduler):
         :param circuit_scheduler (CircuitSchedule): Circuit schedule data
         :return:
         """
-        job_call = None
         if circuit_scheduler.action == 'create':
             job_call = circuit.deploy
         elif circuit_scheduler.action == 'remove':
             job_call = circuit.remove
+        else:
+            return
```

Should this really go silent, or should we raise an exception here?